content/doc/book/blueocean/_blue-ocean-status.adoc

```diff
@@ -0,0 +1,14 @@
+////
+This file is only meant to be included as a snippet in other documents.
+////
+
+[NOTE]
+.Blue Ocean status
+====
+Blue Ocean will not receive further functionality updates.
+Blue Ocean will continue to provide easy-to-use Pipeline visualization, but will not be enhanced further.
```

Comment:
Incorrect grammar: the use of a comma before the coordinating conjunction "but" signifies the joining of two independent clauses, but the second clause is missing a subject:

```suggestion
Blue Ocean will continue to provide easy-to-use Pipeline visualization, but it will not be enhanced further.
```

Reply:
Thanks!  Included in d3b619b847852f97adf44b0001a2b3c8680faf00